charts/kvisor/templates/agent.yaml

```diff
@@ -120,6 +122,11 @@ spec:
               port: metrics
               path: /healthz
             periodSeconds: 5
+          readinessProbe:
```

Comment:
Agent doesn't have any k8s service and doesn't serve traffic. But if this is due to compliance I guess we can leave it.